pkg/kernel/Dockerfile

```diff
@@ -55,7 +58,7 @@ FROM kernel-build-base-native AS kernel-target-arm64-build-arm64
 # set versions for arm64
 # hadolint ignore=DL3006
 FROM kernel-target-${TARGETARCH}-build-${BUILDARCH} AS kernel-target-arm64
-ARG KERNEL_VERSION_arm64=5.10.121
+ARG KERNEL_VERSION_arm64=5.10.131
```

Comment:
Can you do this version bump as a separate commit to make it easier to see in git log?

Reply:
Already in a separate commit: pkg/kernel: bump up minor version of the current stable kernel to 5.1… ￼…